elyra/airflow/bootstrapper.py

```diff
@@ -376,8 +382,13 @@ def package_list_to_dict(cls, filename: str) -> dict:
                         package_name, package_version = line.strip("\n").split(sep=" @ ")
                     elif "===" in line:
                         package_name, package_version = line.strip("\n").split(sep="===")
-                    else:
+                    elif "==" in line:
                         package_name, package_version = line.strip("\n").split(sep="==")
+                    elif "-e" in line:
```

Comment:
this would have unintended consequences if the entry contains the sequence `-e` anywhere, e.g. `package-entry`. See https://github.com/elyra-ai/elyra/blob/master/test_requirements.txt#L1 for an example